handler/handler.go

```diff
@@ -57,6 +60,11 @@ func (batchHandler BatchHandler) sendToES(ctx context.Context, esDestURL string,
 
 	var bulkupsert []byte
 	for _, event := range events {
+		if event.UID == "" {
+			log.Info(ctx, "No uid for inbound kafka event, no transformation possible")
```

Comment:
the message should be all lower-case
`"no uid for inbound kafka event, no transformation possible"`